src/coffea/nanoevents/methods/vector.py

```diff
@@ -43,12 +43,25 @@
 
 """
 import numbers
+from datetime import datetime
 
 import awkward
 import numba
 import numpy
+import pytz
 from dask_awkward import dask_method
 
+from coffea.util import deprecate
+
+_cst = pytz.timezone("US/Central")
+_depttime = _cst.localize(datetime(2024, 6, 30, 11, 59, 59))
+deprecate(
+    "coffea.nanoevents.methods.vector will be removed and replaced with scikit-hep vector. Consider using that package!",
```

Comment:
Maybe we should say that the coffea schemas that are based on vector will internally be migrated to use vector?